components/back-link/src/__snapshots__/test.js.snap

```diff
@@ -1,14 +1,8 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Back Link matches wrapper snapshot: wrapper mount 1`] = `
-.glamor-1,
-[data-glamor-1] {
-  margin-bottom: 15px;
```

Comment:
wondering where this 15px has gone

Reply:
It's further down in the file with a renamed `.glamor-0`. I had to open the file locally to see this quickly as I couldn't find an easy way to see it in this diff preview. This will also be from the `withWhiteSpace` HoC.